src/legacy/state/orders/updaters/utils.ts

```diff
@@ -49,6 +59,10 @@ export function computeOrderSummary({
       // We only have the API order info, let's at least use that
       summary = `Swap ${sellToken} for ${buyToken}`
     }
+
+    if (owner && receiver && receiver !== owner) {
+      summary += ` to ${shortenAddress(receiver)}`
```

Comment:
Does this change anything with regular SWAP/LIMIT orders?

Reply:
I don't think so

Reply:
I don't think so
Tested limit order with recipient and looked good to me